gravitee-apim-console-webui/src/management/api/runtime-logs-v4/runtime-logs-settings/runtime-logs-message-settings/api-runtime-logs-message-settings.component.scss

```diff
@@ -0,0 +1,88 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../../../scss/gio-layout' as gio-layout;
+
+@import './../../../../../components/gio-button-toggle/gio-button-toggle';
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  @include gio-layout.gio-responsive-content-container;
+}
+
+.settings {
+  padding: 24px;
+
+  &__title {
+    margin-bottom: 0;
+  }
+
+  &__subtitle {
+    margin-bottom: 24px;
+    color: mat.get-color-from-palette(gio.$mat-space-palette, 'lighter40');
+    @include mat.typography-level($typography, 'body-1');
+  }
+
+  &__form-control {
+    margin-bottom: 24px;
+    display: flex;
+    flex-direction: column;
+
+    &__toggle {
+      margin-left: 0;
+    }
+
+    &__title {
+      display: flex;
+      justify-items: center;
+      margin-bottom: 0;
+      color: mat.get-color-from-palette(gio.$mat-space-palette, 'default');
+      @include mat.typography-level($typography, 'body-2');
+    }
+
+    &__subtitle {
+      margin-bottom: 16px;
+      color: mat.get-color-from-palette(gio.$mat-space-palette, 'lighter40');
+      @include mat.typography-level($typography, 'body-1');
+    }
+
+    &__form-field {
+      margin-bottom: -1.25em;
+    }
+
+    &__info {
+      display: flex;
+      flex-direction: row;
+      justify-items: center;
+
+      mat-icon {
+        height: 20px;
```

Comment:
Not a deal-breaker: The UX team told me once that we should use multiples of 4 for all padding, margin, heights, etc etc etc